.circleci/config.yml

```diff
@@ -185,4 +193,5 @@ workflows:
       - no-exception-alot-of-projects
       - no-warning-imports-guava
       - no-warning-imports-java-design-patterns
+      - git-diff
```

Comment:
We need to run it after maven execution of commands like "mvn test", "mvn verify", "mvn install"
As during test phase execution we update resource files

Reply:
Changes Revert